### ISSUE TITLE
Forks and github actions

### ISSUE THREAD

**Issue by asmagill** [Hammerspoon/Spoons#235]
@cmsj, I just noticed that my fork of this repository is showing "This branch is 135 commits ahead of Hammerspoon:master." When I look at the actual commit log, it looks like a bunch things (docs, zip files, etc) were committed by the "Spoons Github Bot" on April 19... Have you recently made changes to the automation scripts that should be disabled for forks? Do I need to change anything (after forcing it back in sync) to prevent this from occurring again?

**Comment by cmsj** [comment]
@asmagill yeah all the docs/zips stuff is handled by Actions now, when a push happens (typically when I merge a PR). You can disable Actions in the settings of your forked repo, and I’m wondering if I should hardcore into the Actioms yaml that it only run on this repo.

**Comment by asmagill** [comment]
I found that setting after I forcibly reset my fork's HEAD... 

Yeah, if it's not too hard, I would recommend that you do hardcode it into the yaml so we don't take anyone else by surprise!

**Comment by cmsj** [comment]
It's now set up to only run the workflow in this repo. I'm going to rework that though, so it can run as a CI for forks and still provide linting results, just without the final commit action.
